Add builtin agreements post (#146)
hackclub/hcb-engr: content/posts/builtin-agreements/post.mdx

import post from '@/components/Post';
export default post({
  title: "🖋️ Sign agreements directly on HCB!",
  slug: "builtin-agreements",
  category: "improvement",
  tags: [],
  authors: ["samuel"],
  date: new Date("2026-05-23")
});

You can now sign your fiscal sponsorship agreement without leaving HCB!

> You'll see your agreement along with answers to common questions about the agreement.

Reviewer: ```suggestion
You'll see your agreement alongside answers to commonly asked questions.
```
